src/picongpu/submit/hypnos-hzdr/picongpu.profile.example

```diff
@@ -8,22 +8,22 @@ then
 #       export MODULES_NO_OUTPUT=1
 
         # Core Dependencies
-        module load gcc/4.8.2
+        module load gcc/4.9.2
         module load cmake/3.3.0
-        module load boost/1.60.0
-        module load cuda/6.5
-        module load openmpi/1.8.4.kepler
+        module load boost/1.62.0
+        module load cuda/8.0
+        module load openmpi/1.8.6.kepler.cuda80
 
         # Plugins (optional)
         module load pngwriter/0.5.6
-        module load hdf5-parallel/1.8.14 libsplash/1.6.0
+        module load hdf5-parallel/1.8.15 libsplash/1.6.0
```

Comment:
still setting them up - I just tried to load them now.

let us wait until I get the mail they are officially there (updated description)